package.json

```diff
@@ -61,5 +61,8 @@
     "webpack-cli": "^3.1.2",
     "webpack-dev-server": "^3.1.10"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "peerDependencies": {
```

Comment:
why peerDependencies? Wouldn't devDependencies be better suited since it's only useful for TS typings? Having it in peerDependencies wouldn't increase the overall size while not being actually useful for a production usage?